.github/actions/csv-vulnerability-filter/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+// .github/actions/csv-vulnerability-filter/tsconfig.json
```

Comment:
You can delete this, there should only be one tsconfig.json (at the project root)